### DISCUSSION TITLE
What comes next in this sequence?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1135] [Galaxy Trivia] [answered]
```
elgnairt  erauqs  nogatnep  nogaxeh  nogatpeh ???
```

**Comment by peterbaccaro** [comment]
These are geometric shapes where the number of sides increases and the words are reversed.

- elgnairt = triangle = 3 sides
- erauqs = square = 4 sides
- nogatnep = pentagon = 5 sides
- nogaxeh = hexagon = 6 sides
- nogatpeh = heptagon = 7 sides

There for the next in the sequence is is therefore a shape with 8 sides which is an octagon with the word then reversed:

- **nogatco** - octagon = 8 sides

**Comment by thuongtruong1009** [comment, marked as answer]
Let's reverse the letters in the next word:

elgnairt (training)
erauqs (square)
nogatnep (pentagon)
nogaxeh (hexagon)
nogatpeh (?)
The next word in the sequence, with letters reversed, would be "heptagon."
